web/themes/custom/pineapple/components/_hero-image.twig

```diff
@@ -1,10 +1,13 @@
 {% set content_url = content_url|default('') %}
-<div class="hero-container breakout">
+{% set hero_attributes = attributes|default(create_attribute()).addClass([
+  'hero-container',
+  'breakout'
+]) %}
+<div{{ attributes }}>
   {% if content_url %}
     <a href="{{ content_url }}">
   {% endif %}
-  <div class="hero-image" style="background-image: url({{ image_url }})">
-  </div>
+  <div class="hero-image" style="background-image: url({{ image_url }})"></div>
   <div class="hero-image-text container">
```

Comment:
Ensure that this div only shows one of the fields/variables within it are available

Reply:
Can I do an if statement around it that lists all the variables inside the div?

Reply:
Yes around the whole thing because currently on your author page you don't have any text or subtext, so it means an empty div is being rendered. So you need to add a check to see if one of those variables has content inside them

Reply:
When I add the if statement, it stops the footer from being on the bottom on every page with a hero image.

Reply:
I think you've added the if statement incorrectly it should be like `{% if pretext or title or subtext or labels %}`

Reply:
It is exactly what I did... and interestingly the issue is with the div with author-body class on it.

Reply:
Okay so I'm guessing you've added the closing end if in the wrong place and the browser thinks there's a stray div tag

Reply:
Good guess. It's done, thank you.